mlflow/sklearn/utils.py

```diff
@@ -615,9 +624,19 @@ def _create_child_runs_for_parameter_search(cv_estimator, parent_run, child_tags
     # the seed estimator and update them with parameter subset specified
     # in the result row
     base_params = seed_estimator.get_params(deep=should_log_params_deeply)
-
     cv_results_df = pd.DataFrame.from_dict(cv_estimator.cv_results_)
-    for _, result_row in cv_results_df.iterrows():
+    rank_column_name = "rank_test_score"
+    if rank_column_name not in cv_results_df.columns.values:
+        rank_column_name = first_custom_rank_column(cv_results_df)
+        warnings.warn(
+            "Top {} child runs will be created based on ordering in {} column. ".format(
+                max_hyper_param_runs, rank_column_name,
+            )
+            + "One can choose not to limit the number of child runs created if so desired."
```

Comment:
How can users do this? I don't see where this PR provides that capability. I'd propose introducing `None` as a valid parameter value to satisfy this use case; we can check for `None` here and avoid limiting results.